### PR TITLE
fix(release-notes): special v4 extension release notes

### PR DESCRIPTION
**Pull request by nicolethoen**
Closes: https://github.com/patternfly/patternfly-react/issues/9080

### CODE REVIEW

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -2,6 +2,56 @@
...
 section: developer-resources
 ---
+## 2023.02.1 release notes (2023-05-01)
+- [@patternfly/react-topology@4.93.5](https://www.npmjs.com/package/@patternfly/react-topology/v/4.93.5)
+- [@patternfly/react-user-feedback@1.0.9](https://www.npmjs.com/package/@patternfly/react-user-feedback/v/1.0.9)
+- [@patternfly/quickstarts@2.4.1](https://www.npmjs.com/package/@patternfly/quickstarts/v/2.4.1)
+
+### Topology
+- Cache css-in-js files ([#6](https://github.com/patternfly/react-topology/pull/6))
+- Remove old typedoc deployment action ([#29](https://github.com/patternfly/react-topology/pull/29))
```
Comment (tlabaj) [inline packages/react-docs/RELEASE-NOTES.md:12]: These should all be in passed tense.

**Review by tlabaj — CHANGES_REQUESTED** [review]

File: packages/react-docs/RELEASE-NOTES.md (hunk 1 above)
Comment (tlabaj) [reply packages/react-docs/RELEASE-NOTES.md:12]: can you also please attach issue

Comment (jessiehuff) [reply packages/react-docs/RELEASE-NOTES.md]: Agreed, sent an updated version! Do we typically include the issue though? I thought we just included the PRs?

**Review by jessiehuff — APPROVED** [review]
LGTM!

File: packages/react-docs/RELEASE-NOTES.md (hunk 1 above)
Comment (tlabaj) [reply packages/react-docs/RELEASE-NOTES.md]: @jessiehuff I meant the issue for this PR for tracking.

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -2,6 +2,36 @@
...
+- Updated running and pending icon colors ([#31](https://github.com/patternfly/react-topology/pull/31))
+- Fixed ColaGroupLayout crash on add ([#21](https://github.com/patternfly/react-topology/pull/21))
+- Kept node outline on hover ([#22](https://github.com/patternfly/react-topology/pull/22))
+- Used badgeTextColor when set ([#20](https://github.com/patternfly/react-topology/pull/20))
+- Fixed display edges inside collapsed groups ([#23](https://github.com/patternfly/react-topology/pull/23))
+- Updated to latest patternfly-react topology (4.91.40) ([#7](https://github.com/patternfly/react-topology/pull/7))
+
+### User feedback
+- Began initial Feedback work to remove product specific functionality ([#9](https://github.com/patternfly/react-user-feedback/pull/9))
+- Updated patternfly to latest. ([#19](https://github.com/patternfly/react-user-feedback/pull/19))
```
Comment (tlabaj) [inline packages/react-docs/RELEASE-NOTES.md:23]: PatternFly

File: packages/react-docs/RELEASE-NOTES.md [hunk 3]
```diff
@@ -2,6 +2,36 @@
...
+- Updated documentation ([#37](https://github.com/patternfly/react-user-feedback/pull/37))
+- Fixed issue with docs not building for production. ([#36](https://github.com/patternfly/react-user-feedback/pull/36))
+- Added fix for callback. ([#32](https://github.com/patternfly/react-user-feedback/pull/32))
+- Added PF Image to extension. ([#24](https://github.com/patternfly/react-user-feedback/pull/24))
```
Comment (tlabaj) [inline packages/react-docs/RELEASE-NOTES.md:27]: Added PatternFly image to extension.

File: packages/react-docs/RELEASE-NOTES.md [hunk 4]
```diff
@@ -2,6 +2,36 @@
...
+- Fixed Feedback links ([#14](https://github.com/patternfly/react-user-feedback/pull/14))
```
Comment (tlabaj) [inline packages/react-docs/RELEASE-NOTES.md:28]: Sentence case. "feedback" here.

**Review by tlabaj — CHANGES_REQUESTED** [review]
looks good a few small nits.

**Review by tlabaj — APPROVED** [review]
LGTM!

**Pull request merged**